woo-razorpay.php

```diff
@@ -1127,7 +1127,7 @@ public function getDefaultCheckoutArguments($order)
             return array(
                 'key'          => $this->getSetting('key_id'),
                 'name'         => html_entity_decode(get_bloginfo('name'), ENT_QUOTES),
-                'currency'     => self::INR,
+                'currency'     => $this->getOrderCurrency($order),
```

Comment:
@abdulwahidsharief can we assign this value to a variable and then pass it to the array, just like it has been done for other values?

Reply:
https://github.com/razorpay/razorpay-woocommerce/pull/563/commits/4e648e38ef9c6697fdb5baacc57f4b83a801bab9